### DISCUSSION TITLE
App bar

### DISCUSSION THREAD

**Discussion by Aravindmu** [reflex-dev/reflex#3864] [General]
how to setup a appbar for reflex apps, and how to create a bottom up sheet?

**Comment by Alek99** [comment]
Can you provide an example of app bar? Or what its supposed to look like

**Reply by Aravindmu** [reply]
Like flutter app bar ,to showing app name like that

**Reply by Aravindmu** [reply]
![OIP~3.jpeg](https://github.com/user-attachments/assets/65bf3bb1-047c-4aad-9ae6-9c92fb964053)


How to create app bar, bottom bar,and floating icon in reflex app
